README.md

```diff
@@ -127,92 +126,93 @@ OakMap<K,V> oak = builder.build();
 
 ### OakMap Methods
 You are welcome to take a look on the OakMap's [full API](https://github.com/yahoo/Oak/wiki/Full-API).
-OakMap's API is similar to the ConcurrentNavigableMap API; some non-standard API methods and special cases are discussed below.
+OakMap's API implements the ConcurrentNavigableMap interface. For improved performance, it offers additional non-standard zero-copy API methods that are discussed below.
 
-### OakBuffers
-OakMap provides two types of memory buffers: *OakRBuffer* (read-only) and *OakWBuffer* (read and write). These buffers support the standard Java API for read-only Java ByteBuffers and writable Java ByteBuffers, respectively.
+### OakRBuffers
+OakMap provides read-only memory buffers, *OakRBuffer*` that support the standard Java API for read-only Java ByteBuffers.
```

Comment:
I don't think it is a good idea to withdraw the OakWBuffer interface. This make it difficult to manage (reuse) the memory.